docs/getting-started.md

```diff
@@ -253,9 +253,12 @@ If you would like to run specific functional tests only, you can leverage `ginkg
 command line options as follows (run a specified suite):
 
 ```
-    FUNC_TEST_ARGS='-focus-file=vmi_networking_test' make functest
+    FUNC_TEST_ARGS='-focus-file=vmi_networking' make functest
 ```
 
+Ginkgo's [Location-Based Filtering](https://onsi.github.io/ginkgo/#location-based-filtering) and [Description-Based Filtering](https://onsi.github.io/ginkgo/#description-based-filtering) documentation 
+describe additional helpful options for focused execution that do not require recompilation.
```

Comment:
It could be nice having it as a [note](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):
```suggestion
> [!NOTE]
> Ginkgo's [Location-Based Filtering](https://onsi.github.io/ginkgo/#location-based-filtering) and [Description-Based Filtering](https://onsi.github.io/ginkgo/#description-based-filtering) documentation 
> describe additional helpful options for focused execution that do not require recompilation.
```